.github/workflows/on-prerelease.yml

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: On Pre-Release
+on:
+  workflow_dispatch:
+  # release:
+  #   types:
+  #     - prereleased
+
+env:
+  ARTIFACTS_API_KEY: ${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }} # | base64 # ToDo: For some reason the pipe is not returning the expected value. Do not know why yet.
```

Comment:
I don't think you can run shell commands from the `env` block, hence the pipe doesn't work.